server/settings/publish_plugins.py

```diff
@@ -40,6 +40,7 @@ def extract_jobinfo_overrides_enum():
         {"value": "secondary_pool", "label": "Secondary pool"},
         {"value": "machine_list", "label": "Machine List"},
         {"value": "machine_list_deny", "label": "Machine List is a Deny"},
+        {"value": "publish_job_suspended", "label": "Publish Job Suspended"},
```

Comment:
```suggestion
        {"value": "suspend_publish_job", "label": "Suspend Publish Job"},
```

Reply:
I actually think an enum is better: seeing it then be either Queued or Suspended then it's immediately clear what happens if you do not suspend it.

Reply:
So `publish_job_state` instead?